src/views/RecipeView/RecipeView.js

```diff
@@ -1,11 +1,20 @@
 import React from 'react';
+import recipeData from './recipeData';
+import Recipe from './Recipe';
+import {useParams} from "react-router-dom";
 
 const RecipeView = () => {
     /* todo(przemyslaw.skoczewski)
         Create simple layout with recipe (ingredients list, description, title, image)
      */
+    const data = recipeData.map(recipe => <Recipe key={recipe.id} title={recipe.title}
```

Comment:
You can rename it as well. Here you can also improve some formatting. E.q. you can write it like this:
```javascript
const data = recipeData.map(recipe => 
    <Recipe key={recipe.id} 
        title={recipe.title}
        description={recipe.description} 
        ingredientsList={recipe.ingredientsList} 
        image={recipe.image} path={recipe.id}
    />
)
```